graphrag_sdk/chat_session.py

```
@@ -75,26 +105,37 @@ def __init__(self, model_config: KnowledgeGraphModelConfig, ontology: Ontology,
-            last_answer=self.last_complete_response["response"],
-            cypher_prompt=self.cypher_prompt,
-            cypher_prompt_with_history=self.cypher_prompt_with_history
-        )
-
-        (context, cypher, query_execution_time) = cypher_step.run(message)
+        # Update the last_answer for this query
+        self.cypher_step.last_answer = self.last_complete_response.get("response")
+        
+        try:
+            (context, cypher, query_execution_time) = self.cypher_step.run(message)
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)